pkg/conference/data_channel_message_processor.go

```diff
@@ -28,47 +36,61 @@ func (c *Conference) processSelectDCMessage(participant *Participant, msg event.
 		}
 	}
 
-	// Subscribe to the found tracks.
+	// Subscribe to the found tracks
```

Comment:
OCD: It seems like all other comments have a full-stop at the end, but it was removed for this comment for some reason. Could we get it back? 🙂